src/components/QuestionAdjudication.tsx

```diff
@@ -191,30 +224,59 @@ export function QuestionAdjudication({
   }
 
   const formChanged = () => setCanPublish((prev) => (prev ? !prev : prev))
+
+  const actionButtons = () => {
```

Comment:
Nit: could pull this out into a component in this file and invoke it like a normal component: `<ActionButtons ...props />`

Reply:
Thanks. It's updated.